test/basic-querying.test.js

```diff
@@ -6,6 +6,7 @@
 // This test written in mocha+should.js
 'use strict';
 
+/* global getSchema:false, connectorCapabilities:false */
```

Comment:
I believe it is important to keep our house as clean as possible and as soon as possible. We should not delay these minor issues for months IMO, but I'll leave that decision up to @ritch. Issue created at https://github.com/strongloop/loopback-datasource-juggler/issues/1186 and assigned to @siddhipai